repose-aggregator/components/filters/client-auth/src/main/java/org/openrepose/filters/clientauth/ClientAuthenticationHandlerFactory.java

```diff
@@ -77,7 +78,12 @@ public void configurationUpdated(ClientAuthConfig modifiedConfig) {
                     accountRegexExtractor.addPattern(clientMapping.getIdRegex());
                 }
                 if (modifiedConfig.getAtomFeeds() != null) {
-                    activateOpenstackAtomFeedListener(modifiedConfig);
+                    try {
+                        activateOpenstackAtomFeedListener(modifiedConfig);
+                    } catch (TimeoutException e) {
```

Comment:
The caught `AkkaServiceClientException` is now wrapped in an `UpdateFailedException` and re-thrown. The `UpdateListener` interface was updated so `configurationUpdated()` now throws `UpdateFailedException`. This still bubbles it way up to the `PowerApiConfigurationManager` and `PowerApiUpdateManagerEventListener` just like the NPE used to, only this new exception is logged in a more appropriate manner.